src/themes/helpers.js

```diff
@@ -0,0 +1,42 @@
+function lightenDarkenColor(color, value) {
+  let usePound = false;
+
+  if (color[0] === '#') {
+    color = color.slice(1);
+    usePound = true;
+  }
+
+  const num = parseInt(color, 16);
+
+  // Red
+  let red = (num >> 16) + value;
+  if (red > 255) red = 255;
+  else if (red < 0) red = 0;
+
+  // Green
+  let green = (num & 0x0000FF) + value;
+  if (green > 255) green = 255;
+  else if (green < 0) green = 0;
+
+  // Blue
+  let blue = ((num >> 8) & 0x00FF) + value;
+  if (blue > 255) blue = 255;
+  else if (blue < 0) blue = 0;
+
+  return (usePound?'#':'') + (green | (blue << 8) | (red << 16)).toString(16);
```

Comment:
Please, add spaces here
```suggestion
  return (usePound ? '#' : '') + (green | (blue << 8) | (red << 16)).toString(16);
```